SwrveSDK/SDK/Talk/SwrveMessageController.m

```diff
@@ -790,36 +784,39 @@ -(SwrveMessage*)getMessageForEvent:(NSString *)event
         
         // If QA enabled, send message selection information
         if(self.qaUser != nil) {
-            [self.qaUser trigger:event withMessage:result withReason:campaignReasons withMessages:campaignMessages];
+            [self.qaUser trigger:eventName withMessage:result withReason:campaignReasons withMessages:campaignMessages];
```

Comment:
That would be super useful, or at least mention what didn't match in the 'reason'